ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/ProjectList/Index.cshtml

```diff
@@ -27,7 +77,7 @@
 		<hr class="govuk-section-break govuk-section-break--visible">
 
 		@{
-			var index = 0;
+			var index1 = 0;
 			foreach (var project in Model.Projects)
```

Comment:
Minor: my preference in this case would be to change the foreach to a for loop. But this works as well.

Reply:
There was some unwanted commented code of old mark up in there that I should have removed. I have removed it now. Foreach() gives us the object for individual project but we also need the index for markup IDs. Switching to a For() loop will give us the index but we will have to access individual project via the list index [] which is equally ugly, so I chose to opt for convenience.